JPS_Ontology/ontology/ontoassetmanagement/OntoAssetManagement.csv

```diff
@@ -30,8 +30,38 @@ UnitPrice,Class,Price,IS-A,,,,Price per individual product or service.,https://w
+hasAccount,Object Property,,,ResearchGrant,https://spec.edmcouncil.org/fibo/ontology/FBC/ProductsAndServices/ClientsAndAccounts/Account,,A relationship which describes the account that a research grant is allocated under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has account
+hasBudget,Object Property,,,https://spec.edmcouncil.org/fibo/ontology/FBC/ProductsAndServices/ClientsAndAccounts/Account,Budget,,A relationship which describes the available budget of an account.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has budget
+hasBudgetCategory,Object Property,,,Budget,BudgetCategory,,A relationship which describes the categories under a budget.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has budget category
+hasServiceCode,Object Property,,,BudgetCategory,ServiceCode,,A relationship which describes the service code of a budget category.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has service code
+hasServiceCategory,Object Property,,,BudgetCategory,ServiceCategory,,A relationship which describes the service category of a budget category.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has service category
+purchasedUnder,Object Property,,,https://purl.org/p2p-o/item#Item,BudgetCategory,,A relationship which describes the budget category that an item is purchased under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,purchased under
+hasMaintenanceSchedule,Object Property,,,https://w3id.org/bot#Element,MaintenanceSchedule,,A relationship that describes the maintenance schedule of an asset.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has maintenance schedule
+hasTask,Object Property,,,MaintenanceSchedule,MaintenanceTask,,A relationship that describes the tasks carried out for a maintenance schedule.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has task
+hasInterval,Object Property,,,MaintenanceTask,http://www.w3.org/2006/time#Interval,,A relationship that describes the interval in which an asset is scheduled for maintenance.,https://www.theworldavatar.com/kg/ontoassetmanagement/,has interval
+isPerformedBy,Object Property,,,MaintenanceTask,https://spec.edmcouncil.org/fibo/ontology/FND/Parties/Parties/IndependentParty,,A relationship that describes the party responsible for performing the maintenance.,https://www.theworldavatar.com/kg/ontoassetmanagement/,is performed by
+performedAt,Object Property,,,MaintenanceTask,http://www.w3.org/2006/time#Instant,,A relationship that describes when a maintenance task was last performed.,https://www.theworldavatar.com/kg/ontoassetmanagement/,performed at
+plannedFor,Object Property,,,MaintenanceTask,http://www.w3.org/2006/time#Instant,,A relationship that describes when the next maintenance task will be carried out.,https://www.theworldavatar.com/kg/ontoassetmanagement/,planned for
```

Comment:
`PlannedFor` is actually very generic, as sth can also be planned for a certain asset and not just another point in time. So, if you can think of sth better quickly, maybe rephrasing could help to resolve this ambiguity. Otherwise, not too critical (from my perspective)

Reply:
would scheduledFor be better? Since this will indicate some sort of arrangement or plan for something to take place at a certain point in time.

Reply:
Agree